CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # CHANGELOG
 
 ## 0.5.3dev
+* Adds telemetry collect for `%sqlplot` invocation
```

Comment:
Check the format at contributing. It should start with a [Fix]

Reply:
Gotcha! I guess this is something we should follow after 0.5.2, it seems the format of the previous version are not quite right

Reply:
Yes it's a change we added recently